lib/spack/spack/package.py

```diff
@@ -1441,17 +1441,16 @@ def do_install(self, **kwargs):
             dep_kwargs['explicit'] = False
             dep_kwargs['install_deps'] = False
             for dep in self.spec.traverse(order='post', root=False):
+                if spack.config.get('config:install_missing_compilers', False):
+                    tty.debug('Boostrapping {0} compiler for {1}'.format(
```

Comment:
@becker33 

Minor typo here ("boostrapping" vs. "bootstrapping")

Otherwise this looks perfect

Reply:
Fixed